relay/generate_docs.py

```diff
@@ -0,0 +1,8 @@
+import inspect
+import os
+import workflow
+
+os.system('pydoc3 -w workflow')
```

Comment:
if you're in a python script, can you just run pydoc APIs (rather than the extra call through os.system)?

https://github.com/python/cpython/blob/3.10/Lib/pydoc.py